packages/utils/src/parse-url.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Checks is the url passed as prop routes to external URL resource by checking if it starts with http, https or mailto
+ * @param link string
+ * @returns boolean
+ */
+export const isExternalLink = (link: string) => /^(http|https|mailto):/i.test(link);
```

Comment:
I don't understand the use of `isExternalLink`, or the variable name needs to be improved.

can we just use `isNavigationFromExternalPlatform` instead.

seems like there is a typo in the comment section as well.

Reply:
the goal and functionality of these functions are different.
with isExternalLink we are checking additionally if the route is to external link and not to the some of the app's page. Basically it i used to resolve our current production issue, when you try to navigate through account settings for responsive version, you see reloading the app. It is due to wrong usage of <a> and <Link>.
so this issue has been fixed in this PR.
will fix the typo 👍